lawnchair/src/app/lawnchair/util/ClipboardUtils.kt

```diff
@@ -0,0 +1,45 @@
+package app.lawnchair.util
+
+import android.content.ClipData
+import android.content.ClipboardManager
+import android.content.Context
+import android.widget.Toast
+import com.android.launcher3.R
+
+fun copyToClipboard(
+    context: Context,
+    text: String,
+    label: String = text,
+    toastMessage: String? = context.getString(R.string.copied_toast),
+) {
+    val clipboardManager: ClipboardManager = context.requireSystemService()
+    val clip = ClipData.newPlainText(label, text)
+    clipboardManager.setPrimaryClip(clip)
+    toastMessage?.let {
+        Toast.makeText(
+            context,
+            it,
+            Toast.LENGTH_SHORT,
+        ).show()
+    }
+}
+
+fun String.copyToClipboard(
+    context: Context,
+    label: String = this,
+    toastMessage: String? = context.getString(R.string.copied_toast),
+) {
+    copyToClipboard(
+        context = context,
+        text = this,
+        label = label,
+        toastMessage = toastMessage,
+    )
+}
```

Comment:
Unused now?

Reply:
I just created it so we can use it later on. Maybe I should get rid of the extension functions for now?!

Reply:
Yep, I believe we can just keep used functions.